ewlegal-opinion.sty

```diff
@@ -9,7 +9,8 @@
 %%%%%
 
 %% Formatting the main title
-\renewcommand\maketitle{%
+\renewcommand{\maketitle}
+{%
```

Comment:
we probably don't need the `%` here. i just like to throw one after open brackets. might as well remove all / add it to all since we are unifying the style.

Reply:
I've got no issues with having

```latex
\command
{%
  Multi-line
}
```

as a style. I updated them all (I think) to follow that style.